### DISCUSSION TITLE
[종료] 주차별 추천할 문제를 자유롭게 코멘트로 작성해 주세요!

### DISCUSSION THREAD

**Discussion by ChanhuiSeok** [CodeTest-StudyGroup/Code-Test-Study#212] [문제 추천하기]
## 👋 [상시] 주차별 추천할 문제를 자유롭게 코멘트로 작성해 주세요!
* Code-Test-Study 그룹에서는 매주 정해진 문제 혹은 문제 목록에 있는 것들을 풀고, 풀이를 자유롭게 업로드하면서 코딩테스트 준비를 하는 공간입니다.

* 그래서 모든 분들과 함께 의견을 나누어 매주 풀이할 문제를 정하려고 합니다.
이 Discussion에 코멘트로 **추천할 문제를 링크와 함께 달아주시면** 감사하겠습니다!

* 추천 문제는 백준 온라인 저지 혹은 저지 시스템이 갖추어진 사이트의 문제로 추천 부탁드립니다.
추천하신 문제를 토대로 해서, 매주 진행할 문제 풀이 항목에 추가하도록 하겠습니다.

* **아무나 많은 의견 및 추천 부탁드립니다. 감사합니다.**

---
### **2021년 11월 이후부터 문제 추천은 더 이상 받지 않습니다.**

**Comment by jjangsungwon** [comment]
수 묶기: https://www.acmicpc.net/problem/1744 추천합니다.

**Reply by ChanhuiSeok** [reply]
> 수 묶기: https://www.acmicpc.net/problem/1744 추천합니다.

감사합니다!! 다음 주차 문제로 등록하였습니다.

**Comment by cheonjiwan** [comment]
여왕벌 : https://www.acmicpc.net/problem/10836 추천합니다!

**Reply by ChanhuiSeok** [reply]
> 여왕벌 : https://www.acmicpc.net/problem/10836 추천합니다!
감사합니다!! 다음 주차 문제로 등록하였습니다.

**Comment by jjangsungwon** [comment]
최솟값 찾기: https://www.acmicpc.net/problem/11003 추천합니다!

**Reply by ChanhuiSeok** [reply]
> 최솟값 찾기: https://www.acmicpc.net/problem/11003 추천합니다!
감사합니다!! 다음 주차 문제로 등록하였습니다.

**Comment by kchung1995** [comment]
메이플스토리: https://www.acmicpc.net/problem/20925 추천합니다!
난이도도 적당하고 괜찮은 문제같습니다.

**Reply by cheonjiwan** [reply]
메이플스토리: https://www.acmicpc.net/problem/20925 추천합니다!
감사합니다 다음주 문제로 등록하겠습니다!

**Comment by kchung1995** [comment]
다오와 디지니의 데이트: https://www.acmicpc.net/problem/20553 추천합니다.

**Reply by cheonjiwan** [reply]
다오와 디지니의 데이트: https://www.acmicpc.net/problem/20553 추천합니다.
추천 감사합니다! 다음주 문제로 등록하겠습니다~!~!!

**Comment by kchung1995** [comment]
디스크 컨트롤러: https://programmers.co.kr/learn/courses/30/lessons/42627 추천합니다!

**Reply by cheonjiwan** [reply]
감사합니다!! 다음 주차 문제로 등록하였습니다.

**Comment by atg0831** [comment]
괄호 회전하기: https://programmers.co.kr/learn/courses/30/lessons/76502 추천합니다.

**Reply by ChanhuiSeok** [reply]
감사합니다!! 다음 주차 문제로 등록하였습니다.

**Comment by kchung1995** [comment]
버블 소트: https://www.acmicpc.net/problem/1517 추천합니다.

**Reply by ChanhuiSeok** [reply]
감사합니다!! 다음 주차 문제로 등록하였습니다.

**Comment by kchung1995** [comment]
입실 퇴실: https://programmers.co.kr/learn/courses/30/lessons/86048 추천합니다!
프로그래머스에서 매주 위클리 챌린지라는 이름으로 문제를 하나씩 공개하고 있는데, 이 문제들을 매주 다루어 보면 어떨까 합니다 (https://programmers.co.kr/learn/challenges).

**Reply by ChanhuiSeok** [reply]
좋은 의견 감사드립니다. 반영하도록 하겠습니다 :)

**Comment by kchung1995** [comment]
구간 합 구하기: https://www.acmicpc.net/problem/2042
구간 합 구하기 2: https://www.acmicpc.net/problem/10999 추천합니다.
자주 등장하는 알고리즘은 아니지만, 한 번쯤 풀어두면 좋을 것 같은 문제들입니다.

**Reply by ChanhuiSeok** [reply]
감사합니다!! 다음 주차 문제로 등록하였습니다.

**Comment by kchung1995** [comment]
난민: https://www.acmicpc.net/problem/23090
새로 추가된 문제인데 문제가 괜찮은 것 같아서 추천합니다!

**Reply by ChanhuiSeok** [reply]
감사합니다!! 다음 주차 문제로 등록하였습니다.

**Comment by ChanhuiSeok** [comment]
### **2021년 11월 이후부터 문제 추천은 더 이상 받지 않습니다. 감사합니다.**
